src/Microsoft.Performance.SDK/Processing/ColumnBuilding/ColumnVariantIdentifier.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Performance.SDK.Processing.ColumnBuilding;
+
+/// <summary>
+///     Represents a identifier for a column variant.
+/// </summary>
+/// <param name="Guid">
+///     The unique identifier for the column variant.
+/// </param>
+/// <param name="Name">
+///     The human-readable name of the column variant.
+/// </param>
+public record ColumnVariantIdentifier(Guid Guid, string Name);
```

Comment:
Would it be helpful to add a description to this? I'm thinking the description might be helpful in a tooltip or similar.

Reply:
If we do decide to add a description, perhaps we should change the name from Identifier to Descriptor, or something similar.

Reply:
I like the idea of using descriptor, since technically the Guid is what actually identifies the variant.